rita_common/src/debt_keeper/mod.rs

```diff
@@ -555,6 +559,15 @@ impl DebtKeeper {
     }
 
     /// This updates a neighbor's debt and outputs a DebtAction if one is necessary.
+    /// /// 2.) payment_threshold : This is the ammount at which a router will make a payment. Below this value, the router will not may a payment since
+    ///     a large portion of the payment will be eaten in fees which is not desirable. This is caluculated by
+    ///     gas_price * gas_required * dynamic_fee_multiplier, where dynamic fee multiplier determines how much larger the payment is
+    ///     compared to the fees (fees are essentially gas_price * gas_required). When this is larger, the router pays less often and
+    ///     vice versa
```

Comment:
comment data is good, but should maybe be down on the variables when we declare them? not as a function comment since the function doesn't take these as arguments, also the numbering is odd.

Reply:
done